src/scss/components/_status.scss

```diff
@@ -0,0 +1,122 @@
+// Status icon styles
+.docs-status__Ready {
+    stroke: $sprk-green;
+    fill: $sprk-green;
+    margin-right: 4px;
+}
+
+.docs-status__UnderReview {
+    stroke: #d2752b;
+    fill: #d2752b;
+    margin-right: 4px;
+}
+
+.docs-status__Updated {
+    stroke: $sprk-blue;
+    fill: $sprk-blue;
+    margin-right: 4px;
+}
+
+.docs-status__New {
+    stroke: $sprk-purple;
+    fill: $sprk-purple;
+    margin-right: 4px;
+}
+
+.docs-status__Research {
+    stroke: $sprk-black;
+    fill: $sprk-black;
+    margin-right: 4px;
+}
+
+.docs-status__Build {
+    stroke: $sprk-black;
+    fill: $sprk-black;
+    margin-right: 4px;
+}
+
+.docs-status__LowPriority {
+    stroke: #86BC3D;
+    fill: #86BC3D;
+    margin-right: 4px;
+}
+
+.docs-status__HighPriority {
+    stroke: $sprk-red;
+    fill: $sprk-red;
+    margin-right: 4px;
+}
+
+
+// Status links hover colors
+.docs-link__green:hover {
+    border-bottom: 2px solid $sprk-green;
+}
+.docs-link__yellow:hover {
+    border-bottom: 2px solid #B46417;
+}
+.docs-link__blue:hover {
+    border-bottom: 2px solid $sprk-blue;
+}
+
+
+// Status additional info question mark styles
+.docs-status-question-mark {
+    height: $sprk-icon-m;
+    width: $sprk-icon-m;
+    stroke-width: $sprk-icon-stroke-width;
+    margin-left: 6px;
+}
+.docs-status-question-mark__blue {
```

Comment:
We're on the right track, I would do some renaming all these classes to focus more on the status than the color. You were doing this with the Status classes. Combine some classes too, it seems like you're separately declaring color and underline color in different classes.

# Reasons
- Colors could change one day
- They're more descriptive about what will change in the component
- Things are more consistent in the code, making it easier to read.